worker-message-prioritization-rerouting/pom.xml

```diff
@@ -140,7 +140,7 @@
                     <containerNamePattern>%a-%t</containerNamePattern>
                     <verbose>true</verbose>
                     <images>
-                        <!-- Build a RabbitMQ image with the shovel plugins enabled for testing purposes -->
+                        <!-- Build a RabbitMQ image -->
```

Comment:
The original comment was wrong, this is just running an image, not building one.